### ISSUE TITLE
This repository breaks Quality standard #4

### ISSUE THREAD

**Issue by ShayBox** [RuyiLi/hacktoberfest-akairo-bot#16]
Everyone that's contributed to this project is/will be disqualified for breaking Quality standard 4 "[Something that's clearly an attempt to simply +1 your PR count for October](https://hacktoberfest.digitalocean.com/details)", which this repository clearly attempts to do.

**Comment by RuyiLi** [comment]
There are many repositories specifically made to help beginners get started with making pull requests; this is no exception.

**Comment by ShayBox** [comment]
That's not what this project is made for, this project was specifically made at the beginning of October with a simple skeleton and even explicitly states it's for increasing pr count, it wasn't an existing project looking to expand or a useful project idea looking to get a jump start, it is and will be completely useless after hacktoberfest and will likely be deleted, if not just abandoned.

**Comment by GamesProSeif** [comment]
I'm scared, period

**Comment by RuyiLi** [comment]
Alright then, I've updated the description and README to reflect the actual intentions of this repository. While I'll admit that some of the pull requests made to this repository have been rather low-quality, some of them clearly had time and effort dedicated into them (e.g. #8, #13, #15). While there's nothing I can do about the ones I have already merged, I'll make sure to invalidate low-effort pull requests in the future.

Additionally, I cannot find any section in the rules stating that the project must have existed prior and remain relevant subsequent to the conclusion of Hacktoberfest. From my point of view, one of the points of this event is to encourage new GitHub users to start contributing to open-source projects; while this project was indeed created solely for Hacktoberfest, I believe that it is a perfectly valid first step for beginners.
